README: Add 1.11.0 and 1.10.0 to release history (#25)
instana/envoy-tracing: README.md

## Release History

### 1.11.0 (2025-01-21)

>   * Add support for envoy-proxy 1.32.2 1.32.3 and 1.33.0

Reviewer: This is irrelevant in this repo here. The Envoy tracer based on OpenTracing is only usable until 1.29.x. Please remove this. TIA
```suggestion
```

>   * Removed the agent header check for the `Server` header in the discovery response.

Reviewer: Maybe let's use the same style as the docs team uses in https://www.ibm.com/docs/en/instana-observability/current?topic=tracer-cpp-sensor-nginx-envoy
```suggestion
  * The discovery response is not checked for the agent header `Server` anymore.
```
